pkg/virt-controller/watch/application.go

```diff
@@ -587,9 +586,6 @@ func (vca *VirtControllerApp) initCommon() {
 		golog.Fatal(err)
 	}
 
-	if err := containerdisk.SetLocalDirectory(filepath.Join(vca.ephemeralDiskDir, "container-disk-data")); err != nil {
```

Comment:
I am not sure I would miss this feature but right now removing this means you cannot specify a custom location and we change the default or don't we?

Reply:
oh right. good catch. Seems like I need to untangle the create and set operation.

Reply:
ah, indeed, now I also see that this is not only about creating the dir.